src/NLog/Targets/IJsonSerializer.cs

```diff
@@ -46,7 +46,19 @@ public interface IJsonSerializer
         /// into JSON format.
         /// </summary>
         /// <param name="value">The object to serialize to JSON.</param>
-        /// <returns>Serialized value.</returns>
+        /// <returns>Serialized value (null = Serialize failed).</returns>
         string SerializeObject(object value);
     }
+
+    internal interface IJsonSerializerV2
```

Comment:
Ow a commit has not been added from mobile...

I think it's important to have the interface public, maybe we could move it to the "NLog" namespace? (and mark the other as obsolete)

While the default serialization is good enough for most cases, it should be possible to replace it with e.g. JSON.NET